src/plugins/intel_gpu/src/graph/prior_box.cpp

```diff
@@ -388,7 +388,7 @@ layout prior_box_inst::calc_output_layout(prior_box_node const& node) {
     // Second feature stores the variance of each prior coordinate.
 
     auto output_data_type = input_layout.data_type == data_types::f16 ? data_types::f16 : data_types::f32;
-    if (!node.get_primitive()->output_data_types.empty())
+    if (node.get_primitive()->output_data_types.at(0))
```

Comment:
Can we sure that there will be at least one element for this property?

Reply:
I found it's meaningless to check empty function because it it initializing the output_data_types vector when creating each primitive. Therefore, as in the original code( if(node.get_primitive()->output_data_type) ), it seems to be correct to check whether the data is set to valid value, So I changed this condition as the original code.